inventories/seapath_standalone_definition_example.yml

```diff
@@ -4,46 +4,61 @@ all:
         standalone_machine:
             hosts:
                 machine1:
+
+                    # Ansible variables
                     ansible_host: 10.0.0.2
+                    ansible_user: ansible
+                    ansible_python_interpreter: /usr/bin/python3
+                    ansible_connection: ssh
+                    ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+                    ansible_remote_tmp: /tmp/.ansible/tmp
+
+                    # Disk configuration
                     main_disk: /dev/sda
+
+                    # Main network configuration
                     network_interface: enp1s0
                     ip_addr: "{{ ansible_host }}"
                     subnet: 24
+                    dns_server: 10.10.2.1
+                    gateway_addr: 10.0.0.1
+                    ntp_primary_server: "ntp.midway.ovh"
+                    ntp_secondary_server: "ntp.unice.fr"
+
+                    # Syslog log sending configuration
+                    # Syslog client keys. If not set, syslog will send logs using an unencrypted connection.
+                    syslog_tls_ca: ../keys/syslog_tls_ca.crt
+                    syslog_tls_key: ../keys/syslog_tls_key.pem
+                    syslog_server_ip: "10.10.10.10" # If not set, syslog will not send any logs from network
+                    logstash_server_ip: 10.10.10.10
+
+                    # Other Network configuration
+                    apply_network_config: true # true to apply the network configuration without rebooting
                     ptp_interface: "enp1s0f1" #OPTIONAL PTP Interface
                     ptp_vlanid: 100 #OPTIONAL VlanID for PTP
                     # SRIOV, Optional configuration
                     sriov_driver: 'igb'
                     sriov:
                       'eno12429': 7
-                    sriov_network_pool_interface: eno12429
-                    sriov_network_pool_name: "{{ 'sr-iov-net-' + sriov_network_pool_interface }}"
-    vars:
-        ansible_user: ansible
-        ansible_python_interpreter: /usr/bin/python3
-        ansible_connection: ssh
-        ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
-        dns_server: 10.10.2.1   # Not valid, just for Ansible
-        gateway_addr: 10.0.0.1 # Not valid, just for Ansible
-        apply_network_config: true
-        ntp_primary_server: "ntp.midway.ovh"
-        ntp_secondary_server: "ntp.unice.fr"
-        ansible_remote_tmp: /tmp/.ansible/tmp
-        syslog_server_ip: "10.0.0.10"
-        logstash_server_ip: "10.0.0.11"
-        workqueuemask: "1001"
-        irqmask: "ffeffe"
```

Comment:
This line is not added anywhere in the file, I think you forget it

Reply:
Done